docs/modules/ROOT/pages/cops_minitest.adoc

```diff
@@ -785,3 +785,30 @@ refute_respond_to(self, :do_something)
 === References
 
 * https://minitest.rubystyle.guide#refute-respond-to
+
+== Minitest/UnspecifiedException
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| Yes
+| No
+| 0.10
+| -
+|===
+
+This cop checks for a specified error in `assert_raises`.
+
+=== Examples
+
+[source,ruby]
+----
+# bad
+assert_raises { do_something}
+assert_raises('message') { do_something}
```

Comment:
I think this case can be allowed like RuboCop RSpec.
https://github.com/rubocop-hq/rubocop-rspec/blob/v1.41.0/lib/rubocop/cop/rspec/unspecified_exception.rb#L24-L26

Reply:
Minitest does not accept strings as exception messages, only exception classes - http://docs.seattlerb.org/minitest/Minitest/Assertions.html#method-i-assert_raises

It only accepts string message as last argument.